csrankings-l.csv

```diff
@@ -545,7 +545,7 @@ Lior Shamir,Kansas State University,http://people.cs.ksu.edu/~lshamir,mP-0XmYAAA
 Lior Wolf,Tel Aviv University,http://www.cs.tau.ac.il/~wolf,UbFrXTsAAAAJ
 Liping Shen,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/~shen-lp,NOSCHOLARPAGE
 Lipyeow Lim,University of Hawaii at Manoa,http://www2.hawaii.edu/~lipyeow,_ILTBg0AAAAJ
-Liqiang Nie,Shandong University,http://ir.sdu.edu.cn/~liqiangnie/index.html,yywVMhUAAAAJ
+liqiang Nie,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/nieliqiang,yywVMhUAAAAJ

```

Comment:
First initial needs to be capitalized.

Reply:
Sorry, it is my carelessness. And I will pull a new request, so please merge my new request preferentially.